### PR TITLE
Further work on partner Pca templates

### PR DESCRIPTION
**Pull request by csabadenes**
- add to the project proper/intended fonts to use
- work on non-latin character display issues

### CODE REVIEW

File: Dockerfile-dev [hunk 1]
```diff
@@ -56,3 +66,14 @@ VOLUME "./:/code/"
 WORKDIR /code/
 
 ENV DJANGO_SETTINGS_MODULE EquiTrack.settings.local
+
+#the env variables above are needed for the VISION api
```
Comment (ntrncic) [inline Dockerfile-dev]: please remove these environment variables from commit.

Comment (csabadenes) [reply Dockerfile-dev]: removed them, sorry about that.

**Comment by csabadenes** [comment]
Connects unicef/etools-issues#433

**Comment by robertavram** [comment]
@csabadenes closing this.. I'll give you details in pvt

**Pull request closed without merging**